test/ast-loader.js

```diff
@@ -0,0 +1,28 @@
+const postcss = require('postcss')
```

Comment:
What is this test doing in particular ?

Reply:
It is pseudo loader before `postcss-loader` which transmits postcss AST

Reply:
exercising the previous code path! This needs to be a separate file for the ast-loader.test file to use

Reply:
Understood